hknweb/candidate/templates/candidate/offreq.html

```diff
@@ -60,6 +70,10 @@ <h3>Past Officer Challenges</h3>
         Waiting for response <img src="{% static 'candidate/maybe.png'%}" alt="in progress" class="inline-icon">
         </p>
 
+        <p>
+            Note: Both you and csec need to confirm a challenge for it to be valid.
```

Comment:
Is the officer seeing this or the candidate? If it's the candidate, this needs to be reworded to "Both your officer and csec."

Reply:
It's the officer